DuggaSys/resulted.js

```diff
@@ -304,6 +304,10 @@ function sorttype(t) {
 
 	var col;
 	var dir;
+
+  localStorage.setItem("lena_" + querystring['cid'] + "-" + querystring['coursevers'] + "-sortcol", col);
```

Comment:
I'm unable to use tab to get the correct indentation, fix that.

Reply:
The code has correct indentation when I opened it in Atom, Visual Code and Notepad. So I can't really change it because it isn't wrong for me.